src/core/annotation.js

```diff
@@ -4070,6 +4070,20 @@ class InkAnnotation extends MarkupAnnotation {
     ink.set("Border", [0, 0, 0]);
     ink.set("Rotate", rotation);
 
+    // Line thickness.
+    const bs = new Dict(xref);
+    ink.set("BS", bs);
+    bs.set("W", thickness);
```

Comment:
If you're adding a `BS`-dict, let's give it a proper /Type-entry as well.
```suggestion
    // Line thickness.
    const bs = new Dict(xref);
    ink.set("BS", bs);
    bs.set("Type", Name.get("Border"));
    bs.set("W", thickness);
```

Reply:
I wondered the same and I checked how Acrobat is saving the BS and it doesn't add a Type for the ink case.